src/strategies/rnbw-balance/index.ts

```diff
@@ -0,0 +1,76 @@
+import { formatUnits } from '@ethersproject/units';
+import Multicaller from '../../utils/multicaller';
+
+export const author = 'mystbrent';
+export const version = '0.1.0';
+
+const abi = [
+  {
+    constant: true,
+    inputs: [
+      {
+        internalType: 'address',
+        name: '',
+        type: 'address'
+      }
+    ],
+    name: 'balanceOf',
+    outputs: [{ internalType: 'uint256', name: '', type: 'uint256' }],
+    payable: false,
+    stateMutability: 'view',
+    type: 'function'
+  },
+  {
+    inputs: [],
+    name: 'getCurrentHaloHaloPrice',
+    outputs: [
+      {
+        internalType: 'uint256',
+        name: '',
+        type: 'uint256'
+      }
+    ],
+    stateMutability: 'view',
+    type: 'function'
+  }
+]
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+) {
+  const blockTag = typeof snapshot === 'number' ? snapshot : 'latest';
+
+  const multi = new Multicaller(network, provider, abi, { blockTag });
+
+  multi.call('dsrtPrice', options.token, 'getCurrentHaloHaloPrice');
+
+  addresses.forEeach((address) => {
```

Comment:
Hi @mystbrent Sorry for not mentioning all issue at once, here there is an issue it is not `forEach`? 
Make sure to:
- Update the branch
- Allow Maintainers to update the branch
- Make sure everything is running without any error when you run `npm test --strategy=rnbw-balance`

Thanks

Reply:
Hi @ChaituVR thanks for reviewing. 

It seems that there is an issue with the testing this strategy when the network is in Rinkeby?

Are there any test networks that can be used for the test? 

Please let me know.

Thank you

Reply:
Issue has been resolved on the latest pushed commit

Reply:
@mystbrent Please Allow Maintainers to update the branch https://docs.github.com/assets/images/help/pull_requests/allow-maintainers-to-make-edits-sidebar-checkbox.png

Reply:
Hi @ChaituVR, sorry for the delay the Allow maintainers to update checkbox is not visible from my end. Did some research and found this article:

https://github.community/t/how-can-we-enable-allow-edits-from-maintainers-by-default/2847/5.

I am still looking for a workaround with this.

Thank you.